pages/03.Tutorials/23.using-service-accounts-with-rbac/default.en.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Using service accounts with rbac roles'
+published: true
+taxonomy:
+    tag:
+        - kubernetes
+        - service accounts
+        - rbac
+        - user management
+---
+
+The default MetaKube Cluster comes with one admin account to use.
+
+## Prerequisites
+
+* You need to have [created a MetaKube cluster](../02.create-a-cluster/default.en.md) and [installed and configured kubectl](../07.using-kubectl/default.en.md).
+
+## Creating your first view only service account
+
+```shell
+$ kubectl create serviceaccount metakube
+serviceaccount/metakube created
+```
+
+## Fetch the service account with the service token secret
+
+```shell
+$ kubectl get serviceaccounts metakube -o yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: "2019-02-25T14:59:26Z"
+  name: metakube
+  namespace: default
+  resourceVersion: "3342411"
+  selfLink: /api/v1/namespaces/default/serviceaccounts/metakube
+  uid: f20cfa03-390d-11e9-9906-0a580af40ae2
+secrets:
+- name: metakube-token-n9w5x
+```
+
+### Fetch the users service token
+
+```shell
+$ kubectl get secrets metakube-token-n9w5x -o yaml
+apiVersion: v1
+data:
+  ca.crt: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  namespace: XXXXX==
+  token: bWFuYWdlZDprdWJlcm5ldGVzPW1ldGFrdWJl
+kind: Secret
+metadata:
+  annotations:
+    kubernetes.io/service-account.name: metakube
+    kubernetes.io/service-account.uid: f20cfa03-390d-11e9-9906-0a580af40ae2
+  creationTimestamp: "2019-02-25T14:59:27Z"
+  name: metakube-token-n9w5x
+  namespace: default
+  resourceVersion: "3342410"
+  selfLink: /api/v1/namespaces/default/secrets/metakube-token-n9w5x
+  uid: f21405c0-390d-11e9-9024-0a580af4096f
+type: kubernetes.io/service-account-token
+```
+
+### Copy the base64 encoded token from the secret and decode it
+
+```shell
+$ echo "bWFuYWdlZDprdWJlcm5ldGVzPW1ldGFrdWJl" | base64 --decode
+managed:kubernetes=metakube
+```
+
+### Copy the decoded token to a kube config file
+
+See [Download the kubeconfig](../06.download-the-kubeconfig/default.en.md)
+
+### Add a cluster role binding (cluster wide policy) to the users service account
```

Comment:
This could need some explanation what the cluster role bindind does, like:

"With a ClusterRoleBinding you can configure what permissions the ServiceAccount should have in the cluster. Per default a ServiceAccount has no permissions at all. This example gives the ServiceAccount to only view resources inside of the cluster, but not create, edit or delete them. For more information see Kubernetes Docu".